File: StkExtensionPlugins/Graphics/CSharp/ProjectionRasterStreamPlugin/ProjectionPositionOrientation.cs
using System;
using System.Collections.Generic;
using System.Text;
using AGI.STKUtil;

namespace ProjectionRasterStreamPlugin
{
    public class ProjectionPositionOrientation
    {
        public ProjectionPositionOrientation(Array position, Array rotation)
        {
            m_Position = position;
            m_Orientation = rotation;
        }

        public Array Position
        {
            get { return m_Position; }
            set { m_Position = value; }
        }

        public Array Orientation
        {
            get { return m_Orientation; }
            set { m_Orientation = value; }
        }

        private Array m_Position;
        private Array m_Orientation;
    }
}

File: StkExtensionPlugins/Graphics/VB.Net/ProjectionRasterStreamPlugin/ProjectionPositionOrientation.vb
Imports System.Collections.Generic
Imports System.Text
Imports AGI.STKUtil

Public Class ProjectionPositionOrientation
	Public Sub New(position As Array, rotation As Array)
		m_Position = position
		m_Orientation = rotation
	End Sub

	Public Property Position() As Array
		Get
			Return m_Position
		End Get
		Set
			m_Position = value
		End Set
	End Property

	Public Property Orientation() As Array
		Get
			Return m_Orientation
		End Get
		Set
			m_Orientation = value
		End Set
	End Property

	Private m_Position As Array
	Private m_Orientation As Array
End Class
